### ISSUE TITLE
Questions about Cross-Domain Few-Shot Learning (CD-FSL) Challenge

### ISSUE THREAD

**Issue by leezhp1994** [IBM/cdfsl-benchmark#13]
Hallo, I see in your paper, the results of approach based Transfer learning is obviously better than the results of methods based Meta learning. I wonder if the challenge will evaluate the results of the two kind of methods separately.
And could we use a different backbone networks, such as ResNet18?

**Comment by yunhuiguo** [comment]
Hi @leezhp1994, you can propose your own methods for the challenge, it does not matter  that whether it is transfer learning or meta-learning. And you can use any backbone.

**Comment by chrisyxue** [comment]
Hi, in my opinion, the transfer learning methods shown in your paper actually use the samples from the target domain to fine-tune. However, the CD-FSL challenge requires us only to use the data from mini-imagenet, which means we cannot use any kind of fine-tune. Is that right?

**Comment by yunhuiguo** [comment]
@chrisyxue You can use fine-tune. The data from mini-imagenet is used for pre-training. You can use the support set of the target domain data for fine-tuning.

**Comment by leezhp1994** [comment]
> @chrisyxue You can use fine-tune. The data from mini-imagenet is used for pre-training. You can use the support set of the target domain data for fine-tuning.

So can we use the data from mini-imagenet when fine-tuning?

**Comment by leezhp1994** [comment]
hi @yunhuiguo, can we use the data from mini-imagenet when fine-tuning?

**Comment by yunhuiguo** [comment]
> hi @yunhuiguo, can we use the data from mini-imagenet when fine-tuning?

Yes.

**Comment by CSer-Tang-hao** [comment]
> > hi @yunhuiguo, can we use the data from mini-imagenet when fine-tuning?
> 
> Yes.
 When don't use any ground truth of query data, can we use the pseudo-label of query sets or access the query data when fine-tuning?
